README.md

```diff
@@ -63,3 +71,72 @@ Extract
             console.log(data);
           }
         });
+
+Extract V2
+----
+  Extract metadata from a provided list of URLs.
+
+* **URL**
+
+  http://embedly-proxy.dev.mozaws.net/v2/extract
+
+* **Method:**
+
+  `POST`
+
+*  **URL Params**
+
+  None
+
+* **Data Params**
+
+  * **urls**
+
+    The URLs from which to extract metadata.
+    This should appear as a JSON encoded array of unencoded urls.
+
+    ex: `"["http://www.mozilla.org","https://developer.mozilla.org/en-US/docs/Web/JavaScript"]"`
```

Comment:
@k88hudson Ah if we want to set the `content-encoding` header to `application/json` and JSON encode the entire POST body we can do that too, I just didn't because that's not a default behaviour.